src/platform/forms-system/src/js/components/SchemaForm.jsx

```diff
@@ -237,6 +240,8 @@ SchemaForm.defaultProps = {
   // re: the implicit role:
   // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form
   addNameAttribute: false,
+  // Prevent submit event from propagating up past the form element
+  stopPropagation: false,
```

Comment:
This is kinda tricky, but if this Form -> Modal -> Form situation is the first time that this has become a problem, would it be better to just not have a `stopPropogation` prop and have that be the default behavior?

Reply:
Oops, I forgot to remove this after my recent updates.

Reply:
Fixed. Now there are no changes to the `SchemaForm` component.

Reply:
And with that, the design system team is no longer a required approver :tada: